<?xml version="1.0" encoding="UTF-8" ?>
<testsuites>
  <testsuite errors="1" failures="4" hostname="ROGG46" id="0" name="AnalistaControllerTests" package="com.climaescolar" tests="8" time="3.701" timestamp="2015-07-23T05:50:12">
      <properties />

      <testcase classname="com.climaescolar.AnalistaControllerTests" name="testUpdate" time="2.434">
          <failure message="Assertion failed: &#xa;&#xa;assert analista.save() != null&#xa;       |        |      |&#xa;       |        null   false&#xa;       com.climaescolar.Analista : (unsaved)&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert analista.save() != null
       |        |      |
       |        null   false
       com.climaescolar.Analista : (unsaved)

	at com.climaescolar.AnalistaControllerTests.testUpdate(AnalistaControllerTests.groovy:100)
</failure>

      </testcase>

      <testcase classname="com.climaescolar.AnalistaControllerTests" name="testEdit" time="0.1">
          <failure message="Assertion failed: &#xa;&#xa;assert analista.save() != null&#xa;       |        |      |&#xa;       |        null   false&#xa;       com.climaescolar.Analista : (unsaved)&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert analista.save() != null
       |        |      |
       |        null   false
       com.climaescolar.Analista : (unsaved)

	at com.climaescolar.AnalistaControllerTests.testEdit(AnalistaControllerTests.groovy:80)
</failure>

      </testcase>

      <testcase classname="com.climaescolar.AnalistaControllerTests" name="testDelete" time="0.078">
          <failure message="Assertion failed: &#xa;&#xa;assert analista.save() != null&#xa;       |        |      |&#xa;       |        null   false&#xa;       com.climaescolar.Analista : (unsaved)&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert analista.save() != null
       |        |      |
       |        null   false
       com.climaescolar.Analista : (unsaved)

	at com.climaescolar.AnalistaControllerTests.testDelete(AnalistaControllerTests.groovy:144)
</failure>

      </testcase>

      <testcase classname="com.climaescolar.AnalistaControllerTests" name="testShow" time="0.093">
          <failure message="Assertion failed: &#xa;&#xa;assert analista.save() != null&#xa;       |        |      |&#xa;       |        null   false&#xa;       com.climaescolar.Analista : (unsaved)&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert analista.save() != null
       |        |      |
       |        null   false
       com.climaescolar.Analista : (unsaved)

	at com.climaescolar.AnalistaControllerTests.testShow(AnalistaControllerTests.groovy:62)
</failure>

      </testcase>

      <testcase classname="com.climaescolar.AnalistaControllerTests" name="testCreate" time="0.077" />

      <testcase classname="com.climaescolar.AnalistaControllerTests" name="testList" time="0.61" />

      <testcase classname="com.climaescolar.AnalistaControllerTests" name="testIndex" time="0.061" />

      <testcase classname="com.climaescolar.AnalistaControllerTests" name="testSave" time="0.159">
          <error message="No signature of method: com.climaescolar.Role.methodMissing() is applicable for argument types: () values: []" type="groovy.lang.MissingMethodException">groovy.lang.MissingMethodException: No signature of method: com.climaescolar.Role.methodMissing() is applicable for argument types: () values: []
	at com.climaescolar.AnalistaController.save(AnalistaController.groovy:27)
	at com.climaescolar.AnalistaControllerTests.testSave(AnalistaControllerTests.groovy:38)
</error>

      </testcase>

      <system-out><![CDATA[--Output from testUpdate--
--Output from testEdit--
--Output from testDelete--
--Output from testShow--
--Output from testCreate--
--Output from testList--
--Output from testIndex--
--Output from testSave--
]]></system-out>

      <system-err><![CDATA[--Output from testUpdate--
--Output from testEdit--
--Output from testDelete--
--Output from testShow--
--Output from testCreate--
--Output from testList--
--Output from testIndex--
--Output from testSave--
]]></system-err>

  </testsuite>
  <testsuite errors="0" failures="1" hostname="ROGG46" id="1" name="AnalistaTests" package="com.climaescolar" tests="1" time="0.223" timestamp="2015-07-23T05:50:15">
      <properties />

      <testcase classname="com.climaescolar.AnalistaTests" name="testSomething" time="0.094">
          <failure message="junit.framework.AssertionFailedError: Implement me" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: Implement me
	at com.climaescolar.AnalistaTests.testSomething(AnalistaTests.groovy:15)
</failure>

      </testcase>

      <system-out><![CDATA[--Output from testSomething--
]]></system-out>

      <system-err><![CDATA[--Output from testSomething--
]]></system-err>

  </testsuite>
  <testsuite errors="0" failures="5" hostname="ROGG46" id="2" name="ColegioControllerTests" package="com.climaescolar" tests="8" time="1.001" timestamp="2015-07-23T05:50:16">
      <properties />

      <testcase classname="com.climaescolar.ColegioControllerTests" name="testUpdate" time="0.374">
          <failure message="Assertion failed: &#xa;&#xa;assert colegio.save() != null&#xa;       |       |      |&#xa;       |       null   false&#xa;       com.climaescolar.Colegio : (unsaved)&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert colegio.save() != null
       |       |      |
       |       null   false
       com.climaescolar.Colegio : (unsaved)

	at com.climaescolar.ColegioControllerTests.testUpdate(ColegioControllerTests.groovy:100)
</failure>

      </testcase>

      <testcase classname="com.climaescolar.ColegioControllerTests" name="testEdit" time="0.085">
          <failure message="Assertion failed: &#xa;&#xa;assert colegio.save() != null&#xa;       |       |      |&#xa;       |       null   false&#xa;       com.climaescolar.Colegio : (unsaved)&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert colegio.save() != null
       |       |      |
       |       null   false
       com.climaescolar.Colegio : (unsaved)

	at com.climaescolar.ColegioControllerTests.testEdit(ColegioControllerTests.groovy:80)
</failure>

      </testcase>

      <testcase classname="com.climaescolar.ColegioControllerTests" name="testDelete" time="0.063">
          <failure message="Assertion failed: &#xa;&#xa;assert colegio.save() != null&#xa;       |       |      |&#xa;       |       null   false&#xa;       com.climaescolar.Colegio : (unsaved)&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert colegio.save() != null
       |       |      |
       |       null   false
       com.climaescolar.Colegio : (unsaved)

	at com.climaescolar.ColegioControllerTests.testDelete(ColegioControllerTests.groovy:144)
</failure>

      </testcase>

      <testcase classname="com.climaescolar.ColegioControllerTests" name="testShow" time="0.061">
          <failure message="Assertion failed: &#xa;&#xa;assert colegio.save() != null&#xa;       |       |      |&#xa;       |       null   false&#xa;       com.climaescolar.Colegio : (unsaved)&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert colegio.save() != null
       |       |      |
       |       null   false
       com.climaescolar.Colegio : (unsaved)

	at com.climaescolar.ColegioControllerTests.testShow(ColegioControllerTests.groovy:62)
</failure>

      </testcase>

      <testcase classname="com.climaescolar.ColegioControllerTests" name="testCreate" time="0.056" />

      <testcase classname="com.climaescolar.ColegioControllerTests" name="testList" time="0.062" />

      <testcase classname="com.climaescolar.ColegioControllerTests" name="testIndex" time="0.056" />

      <testcase classname="com.climaescolar.ColegioControllerTests" name="testSave" time="0.088">
          <failure message="Assertion failed: &#xa;&#xa;assert response.redirectedUrl == &apos;/colegio/show/1&apos;&#xa;       |        |             |&#xa;       |        null          false&#xa;       org.codehaus.groovy.grails.plugins.testing.GrailsMockHttpServletResponse@2d6ae9c2&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert response.redirectedUrl == &apos;/colegio/show/1&apos;
       |        |             |
       |        null          false
       org.codehaus.groovy.grails.plugins.testing.GrailsMockHttpServletResponse@2d6ae9c2

	at com.climaescolar.ColegioControllerTests.testSave(ColegioControllerTests.groovy:48)
</failure>

      </testcase>

      <system-out><![CDATA[--Output from testUpdate--
--Output from testEdit--
--Output from testDelete--
--Output from testShow--
--Output from testCreate--
--Output from testList--
--Output from testIndex--
--Output from testSave--
]]></system-out>

      <system-err><![CDATA[--Output from testUpdate--
--Output from testEdit--
--Output from testDelete--
--Output from testShow--
--Output from testCreate--
--Output from testList--
--Output from testIndex--
--Output from testSave--
]]></system-err>

  </testsuite>
  <testsuite errors="1" failures="4" hostname="ROGG46" id="3" name="CordinadorControllerTests" package="com.climaescolar" tests="8" time="0.875" timestamp="2015-07-23T05:50:17">
      <properties />

      <testcase classname="com.climaescolar.CordinadorControllerTests" name="testUpdate" time="0.346">
          <failure message="Assertion failed: &#xa;&#xa;assert cordinador.save() != null&#xa;       |          |      |&#xa;       |          null   false&#xa;       com.climaescolar.Cordinador : (unsaved)&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert cordinador.save() != null
       |          |      |
       |          null   false
       com.climaescolar.Cordinador : (unsaved)

	at com.climaescolar.CordinadorControllerTests.testUpdate(CordinadorControllerTests.groovy:100)
</failure>

      </testcase>

      <testcase classname="com.climaescolar.CordinadorControllerTests" name="testEdit" time="0.081">
          <failure message="Assertion failed: &#xa;&#xa;assert cordinador.save() != null&#xa;       |          |      |&#xa;       |          null   false&#xa;       com.climaescolar.Cordinador : (unsaved)&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert cordinador.save() != null
       |          |      |
       |          null   false
       com.climaescolar.Cordinador : (unsaved)

	at com.climaescolar.CordinadorControllerTests.testEdit(CordinadorControllerTests.groovy:80)
</failure>

      </testcase>

      <testcase classname="com.climaescolar.CordinadorControllerTests" name="testDelete" time="0.067">
          <failure message="Assertion failed: &#xa;&#xa;assert cordinador.save() != null&#xa;       |          |      |&#xa;       |          null   false&#xa;       com.climaescolar.Cordinador : (unsaved)&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert cordinador.save() != null
       |          |      |
       |          null   false
       com.climaescolar.Cordinador : (unsaved)

	at com.climaescolar.CordinadorControllerTests.testDelete(CordinadorControllerTests.groovy:144)
</failure>

      </testcase>

      <testcase classname="com.climaescolar.CordinadorControllerTests" name="testShow" time="0.057">
          <failure message="Assertion failed: &#xa;&#xa;assert cordinador.save() != null&#xa;       |          |      |&#xa;       |          null   false&#xa;       com.climaescolar.Cordinador : (unsaved)&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert cordinador.save() != null
       |          |      |
       |          null   false
       com.climaescolar.Cordinador : (unsaved)

	at com.climaescolar.CordinadorControllerTests.testShow(CordinadorControllerTests.groovy:62)
</failure>

      </testcase>

      <testcase classname="com.climaescolar.CordinadorControllerTests" name="testCreate" time="0.051" />

      <testcase classname="com.climaescolar.CordinadorControllerTests" name="testList" time="0.053" />

      <testcase classname="com.climaescolar.CordinadorControllerTests" name="testIndex" time="0.049" />

      <testcase classname="com.climaescolar.CordinadorControllerTests" name="testSave" time="0.061">
          <error message="No signature of method: com.climaescolar.Role.methodMissing() is applicable for argument types: () values: []" type="groovy.lang.MissingMethodException">groovy.lang.MissingMethodException: No signature of method: com.climaescolar.Role.methodMissing() is applicable for argument types: () values: []
	at com.climaescolar.CordinadorController.save(CordinadorController.groovy:27)
	at com.climaescolar.CordinadorControllerTests.testSave(CordinadorControllerTests.groovy:38)
</error>

      </testcase>

      <system-out><![CDATA[--Output from testUpdate--
--Output from testEdit--
--Output from testDelete--
--Output from testShow--
--Output from testCreate--
--Output from testList--
--Output from testIndex--
--Output from testSave--
]]></system-out>

      <system-err><![CDATA[--Output from testUpdate--
--Output from testEdit--
--Output from testDelete--
--Output from testShow--
--Output from testCreate--
--Output from testList--
--Output from testIndex--
--Output from testSave--
]]></system-err>

  </testsuite>
  <testsuite errors="0" failures="1" hostname="ROGG46" id="4" name="CordinadorTests" package="com.climaescolar" tests="1" time="0.216" timestamp="2015-07-23T05:50:18">
      <properties />

      <testcase classname="com.climaescolar.CordinadorTests" name="testSomething" time="0.074">
          <failure message="junit.framework.AssertionFailedError: Implement me" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: Implement me
	at com.climaescolar.CordinadorTests.testSomething(CordinadorTests.groovy:15)
</failure>

      </testcase>

      <system-out><![CDATA[--Output from testSomething--
]]></system-out>

      <system-err><![CDATA[--Output from testSomething--
]]></system-err>

  </testsuite>
  <testsuite errors="0" failures="5" hostname="ROGG46" id="5" name="CursoControllerTests" package="com.climaescolar" tests="8" time="0.857" timestamp="2015-07-23T05:50:18">
      <properties />

      <testcase classname="com.climaescolar.CursoControllerTests" name="testUpdate" time="0.374">
          <failure message="Assertion failed: &#xa;&#xa;assert curso.save() != null&#xa;       |     |      |&#xa;       |     null   false&#xa;       com.climaescolar.Curso : (unsaved)&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert curso.save() != null
       |     |      |
       |     null   false
       com.climaescolar.Curso : (unsaved)

	at com.climaescolar.CursoControllerTests.testUpdate(CursoControllerTests.groovy:100)
</failure>

      </testcase>

      <testcase classname="com.climaescolar.CursoControllerTests" name="testEdit" time="0.053">
          <failure message="Assertion failed: &#xa;&#xa;assert curso.save() != null&#xa;       |     |      |&#xa;       |     null   false&#xa;       com.climaescolar.Curso : (unsaved)&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert curso.save() != null
       |     |      |
       |     null   false
       com.climaescolar.Curso : (unsaved)

	at com.climaescolar.CursoControllerTests.testEdit(CursoControllerTests.groovy:80)
</failure>

      </testcase>

      <testcase classname="com.climaescolar.CursoControllerTests" name="testDelete" time="0.056">
          <failure message="Assertion failed: &#xa;&#xa;assert curso.save() != null&#xa;       |     |      |&#xa;       |     null   false&#xa;       com.climaescolar.Curso : (unsaved)&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert curso.save() != null
       |     |      |
       |     null   false
       com.climaescolar.Curso : (unsaved)

	at com.climaescolar.CursoControllerTests.testDelete(CursoControllerTests.groovy:144)
</failure>

      </testcase>

      <testcase classname="com.climaescolar.CursoControllerTests" name="testShow" time="0.043">
          <failure message="Assertion failed: &#xa;&#xa;assert curso.save() != null&#xa;       |     |      |&#xa;       |     null   false&#xa;       com.climaescolar.Curso : (unsaved)&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert curso.save() != null
       |     |      |
       |     null   false
       com.climaescolar.Curso : (unsaved)

	at com.climaescolar.CursoControllerTests.testShow(CursoControllerTests.groovy:62)
</failure>

      </testcase>

      <testcase classname="com.climaescolar.CursoControllerTests" name="testCreate" time="0.038" />

      <testcase classname="com.climaescolar.CursoControllerTests" name="testList" time="0.039" />

      <testcase classname="com.climaescolar.CursoControllerTests" name="testIndex" time="0.038" />

      <testcase classname="com.climaescolar.CursoControllerTests" name="testSave" time="0.051">
          <failure message="Assertion failed: &#xa;&#xa;assert response.redirectedUrl == &apos;/curso/show/1&apos;&#xa;       |        |             |&#xa;       |        null          false&#xa;       org.codehaus.groovy.grails.plugins.testing.GrailsMockHttpServletResponse@2fac4657&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert response.redirectedUrl == &apos;/curso/show/1&apos;
       |        |             |
       |        null          false
       org.codehaus.groovy.grails.plugins.testing.GrailsMockHttpServletResponse@2fac4657

	at com.climaescolar.CursoControllerTests.testSave(CursoControllerTests.groovy:48)
</failure>

      </testcase>

      <system-out><![CDATA[--Output from testUpdate--
--Output from testEdit--
--Output from testDelete--
--Output from testShow--
--Output from testCreate--
--Output from testList--
--Output from testIndex--
--Output from testSave--
]]></system-out>

      <system-err><![CDATA[--Output from testUpdate--
--Output from testEdit--
--Output from testDelete--
--Output from testShow--
--Output from testCreate--
--Output from testList--
--Output from testIndex--
--Output from testSave--
]]></system-err>

  </testsuite>
  <testsuite errors="0" failures="5" hostname="ROGG46" id="6" name="EncuestaControllerTests" package="com.climaescolar" tests="8" time="0.772" timestamp="2015-07-23T05:50:19">
      <properties />

      <testcase classname="com.climaescolar.EncuestaControllerTests" name="testUpdate" time="0.296">
          <failure message="Assertion failed: &#xa;&#xa;assert encuesta.save() != null&#xa;       |        |      |&#xa;       |        null   false&#xa;       com.climaescolar.Encuesta : (unsaved)&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert encuesta.save() != null
       |        |      |
       |        null   false
       com.climaescolar.Encuesta : (unsaved)

	at com.climaescolar.EncuestaControllerTests.testUpdate(EncuestaControllerTests.groovy:100)
</failure>

      </testcase>

      <testcase classname="com.climaescolar.EncuestaControllerTests" name="testEdit" time="0.052">
          <failure message="Assertion failed: &#xa;&#xa;assert encuesta.save() != null&#xa;       |        |      |&#xa;       |        null   false&#xa;       com.climaescolar.Encuesta : (unsaved)&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert encuesta.save() != null
       |        |      |
       |        null   false
       com.climaescolar.Encuesta : (unsaved)

	at com.climaescolar.EncuestaControllerTests.testEdit(EncuestaControllerTests.groovy:80)
</failure>

      </testcase>

      <testcase classname="com.climaescolar.EncuestaControllerTests" name="testDelete" time="0.042">
          <failure message="Assertion failed: &#xa;&#xa;assert encuesta.save() != null&#xa;       |        |      |&#xa;       |        null   false&#xa;       com.climaescolar.Encuesta : (unsaved)&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert encuesta.save() != null
       |        |      |
       |        null   false
       com.climaescolar.Encuesta : (unsaved)

	at com.climaescolar.EncuestaControllerTests.testDelete(EncuestaControllerTests.groovy:144)
</failure>

      </testcase>

      <testcase classname="com.climaescolar.EncuestaControllerTests" name="testShow" time="0.046">
          <failure message="Assertion failed: &#xa;&#xa;assert encuesta.save() != null&#xa;       |        |      |&#xa;       |        null   false&#xa;       com.climaescolar.Encuesta : (unsaved)&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert encuesta.save() != null
       |        |      |
       |        null   false
       com.climaescolar.Encuesta : (unsaved)

	at com.climaescolar.EncuestaControllerTests.testShow(EncuestaControllerTests.groovy:62)
</failure>

      </testcase>

      <testcase classname="com.climaescolar.EncuestaControllerTests" name="testCreate" time="0.06" />

      <testcase classname="com.climaescolar.EncuestaControllerTests" name="testList" time="0.039" />

      <testcase classname="com.climaescolar.EncuestaControllerTests" name="testIndex" time="0.038" />

      <testcase classname="com.climaescolar.EncuestaControllerTests" name="testSave" time="0.045">
          <failure message="Assertion failed: &#xa;&#xa;assert response.redirectedUrl == &apos;/encuesta/show/1&apos;&#xa;       |        |             |&#xa;       |        null          false&#xa;       org.codehaus.groovy.grails.plugins.testing.GrailsMockHttpServletResponse@3efa1662&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert response.redirectedUrl == &apos;/encuesta/show/1&apos;
       |        |             |
       |        null          false
       org.codehaus.groovy.grails.plugins.testing.GrailsMockHttpServletResponse@3efa1662

	at com.climaescolar.EncuestaControllerTests.testSave(EncuestaControllerTests.groovy:48)
</failure>

      </testcase>

      <system-out><![CDATA[--Output from testUpdate--
--Output from testEdit--
--Output from testDelete--
--Output from testShow--
--Output from testCreate--
--Output from testList--
--Output from testIndex--
--Output from testSave--
]]></system-out>

      <system-err><![CDATA[--Output from testUpdate--
--Output from testEdit--
--Output from testDelete--
--Output from testShow--
--Output from testCreate--
--Output from testList--
--Output from testIndex--
--Output from testSave--
]]></system-err>

  </testsuite>
  <testsuite errors="0" failures="5" hostname="ROGG46" id="7" name="GradoControllerTests" package="com.climaescolar" tests="8" time="0.753" timestamp="2015-07-23T05:50:19">
      <properties />

      <testcase classname="com.climaescolar.GradoControllerTests" name="testUpdate" time="0.274">
          <failure message="Assertion failed: &#xa;&#xa;assert grado.save() != null&#xa;       |     |      |&#xa;       |     null   false&#xa;       com.climaescolar.Grado : (unsaved)&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert grado.save() != null
       |     |      |
       |     null   false
       com.climaescolar.Grado : (unsaved)

	at com.climaescolar.GradoControllerTests.testUpdate(GradoControllerTests.groovy:100)
</failure>

      </testcase>

      <testcase classname="com.climaescolar.GradoControllerTests" name="testEdit" time="0.05">
          <failure message="Assertion failed: &#xa;&#xa;assert grado.save() != null&#xa;       |     |      |&#xa;       |     null   false&#xa;       com.climaescolar.Grado : (unsaved)&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert grado.save() != null
       |     |      |
       |     null   false
       com.climaescolar.Grado : (unsaved)

	at com.climaescolar.GradoControllerTests.testEdit(GradoControllerTests.groovy:80)
</failure>

      </testcase>

      <testcase classname="com.climaescolar.GradoControllerTests" name="testDelete" time="0.045">
          <failure message="Assertion failed: &#xa;&#xa;assert grado.save() != null&#xa;       |     |      |&#xa;       |     null   false&#xa;       com.climaescolar.Grado : (unsaved)&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert grado.save() != null
       |     |      |
       |     null   false
       com.climaescolar.Grado : (unsaved)

	at com.climaescolar.GradoControllerTests.testDelete(GradoControllerTests.groovy:144)
</failure>

      </testcase>

      <testcase classname="com.climaescolar.GradoControllerTests" name="testShow" time="0.049">
          <failure message="Assertion failed: &#xa;&#xa;assert grado.save() != null&#xa;       |     |      |&#xa;       |     null   false&#xa;       com.climaescolar.Grado : (unsaved)&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert grado.save() != null
       |     |      |
       |     null   false
       com.climaescolar.Grado : (unsaved)

	at com.climaescolar.GradoControllerTests.testShow(GradoControllerTests.groovy:62)
</failure>

      </testcase>

      <testcase classname="com.climaescolar.GradoControllerTests" name="testCreate" time="0.04" />

      <testcase classname="com.climaescolar.GradoControllerTests" name="testList" time="0.037" />

      <testcase classname="com.climaescolar.GradoControllerTests" name="testIndex" time="0.038" />

      <testcase classname="com.climaescolar.GradoControllerTests" name="testSave" time="0.059">
          <failure message="Assertion failed: &#xa;&#xa;assert response.redirectedUrl == &apos;/grado/show/1&apos;&#xa;       |        |             |&#xa;       |        null          false&#xa;       org.codehaus.groovy.grails.plugins.testing.GrailsMockHttpServletResponse@40b351a0&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert response.redirectedUrl == &apos;/grado/show/1&apos;
       |        |             |
       |        null          false
       org.codehaus.groovy.grails.plugins.testing.GrailsMockHttpServletResponse@40b351a0

	at com.climaescolar.GradoControllerTests.testSave(GradoControllerTests.groovy:48)
</failure>

      </testcase>

      <system-out><![CDATA[--Output from testUpdate--
--Output from testEdit--
--Output from testDelete--
--Output from testShow--
--Output from testCreate--
--Output from testList--
--Output from testIndex--
--Output from testSave--
]]></system-out>

      <system-err><![CDATA[--Output from testUpdate--
--Output from testEdit--
--Output from testDelete--
--Output from testShow--
--Output from testCreate--
--Output from testList--
--Output from testIndex--
--Output from testSave--
]]></system-err>

  </testsuite>
  <testsuite errors="0" failures="5" hostname="ROGG46" id="8" name="HojaDeRespuestasControllerTests" package="com.climaescolar" tests="8" time="0.742" timestamp="2015-07-23T05:50:20">
      <properties />

      <testcase classname="com.climaescolar.HojaDeRespuestasControllerTests" name="testUpdate" time="0.298">
          <failure message="Assertion failed: &#xa;&#xa;assert hojaDeRespuestas.save() != null&#xa;       |                |      |&#xa;       |                null   false&#xa;       com.climaescolar.HojaDeRespuestas : (unsaved)&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert hojaDeRespuestas.save() != null
       |                |      |
       |                null   false
       com.climaescolar.HojaDeRespuestas : (unsaved)

	at com.climaescolar.HojaDeRespuestasControllerTests.testUpdate(HojaDeRespuestasControllerTests.groovy:100)
</failure>

      </testcase>

      <testcase classname="com.climaescolar.HojaDeRespuestasControllerTests" name="testEdit" time="0.046">
          <failure message="Assertion failed: &#xa;&#xa;assert hojaDeRespuestas.save() != null&#xa;       |                |      |&#xa;       |                null   false&#xa;       com.climaescolar.HojaDeRespuestas : (unsaved)&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert hojaDeRespuestas.save() != null
       |                |      |
       |                null   false
       com.climaescolar.HojaDeRespuestas : (unsaved)

	at com.climaescolar.HojaDeRespuestasControllerTests.testEdit(HojaDeRespuestasControllerTests.groovy:80)
</failure>

      </testcase>

      <testcase classname="com.climaescolar.HojaDeRespuestasControllerTests" name="testDelete" time="0.039">
          <failure message="Assertion failed: &#xa;&#xa;assert hojaDeRespuestas.save() != null&#xa;       |                |      |&#xa;       |                null   false&#xa;       com.climaescolar.HojaDeRespuestas : (unsaved)&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert hojaDeRespuestas.save() != null
       |                |      |
       |                null   false
       com.climaescolar.HojaDeRespuestas : (unsaved)

	at com.climaescolar.HojaDeRespuestasControllerTests.testDelete(HojaDeRespuestasControllerTests.groovy:144)
</failure>

      </testcase>

      <testcase classname="com.climaescolar.HojaDeRespuestasControllerTests" name="testShow" time="0.061">
          <failure message="Assertion failed: &#xa;&#xa;assert hojaDeRespuestas.save() != null&#xa;       |                |      |&#xa;       |                null   false&#xa;       com.climaescolar.HojaDeRespuestas : (unsaved)&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert hojaDeRespuestas.save() != null
       |                |      |
       |                null   false
       com.climaescolar.HojaDeRespuestas : (unsaved)

	at com.climaescolar.HojaDeRespuestasControllerTests.testShow(HojaDeRespuestasControllerTests.groovy:62)
</failure>

      </testcase>

      <testcase classname="com.climaescolar.HojaDeRespuestasControllerTests" name="testCreate" time="0.037" />

      <testcase classname="com.climaescolar.HojaDeRespuestasControllerTests" name="testList" time="0.041" />

      <testcase classname="com.climaescolar.HojaDeRespuestasControllerTests" name="testIndex" time="0.035" />

      <testcase classname="com.climaescolar.HojaDeRespuestasControllerTests" name="testSave" time="0.041">
          <failure message="Assertion failed: &#xa;&#xa;assert response.redirectedUrl == &apos;/hojaDeRespuestas/show/1&apos;&#xa;       |        |             |&#xa;       |        null          false&#xa;       org.codehaus.groovy.grails.plugins.testing.GrailsMockHttpServletResponse@3129d059&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert response.redirectedUrl == &apos;/hojaDeRespuestas/show/1&apos;
       |        |             |
       |        null          false
       org.codehaus.groovy.grails.plugins.testing.GrailsMockHttpServletResponse@3129d059

	at com.climaescolar.HojaDeRespuestasControllerTests.testSave(HojaDeRespuestasControllerTests.groovy:48)
</failure>

      </testcase>

      <system-out><![CDATA[--Output from testUpdate--
--Output from testEdit--
--Output from testDelete--
--Output from testShow--
--Output from testCreate--
--Output from testList--
--Output from testIndex--
--Output from testSave--
]]></system-out>

      <system-err><![CDATA[--Output from testUpdate--
--Output from testEdit--
--Output from testDelete--
--Output from testShow--
--Output from testCreate--
--Output from testList--
--Output from testIndex--
--Output from testSave--
]]></system-err>

  </testsuite>
  <testsuite errors="0" failures="5" hostname="ROGG46" id="9" name="JornadaControllerTests" package="com.climaescolar" tests="8" time="0.777" timestamp="2015-07-23T05:50:21">
      <properties />

      <testcase classname="com.climaescolar.JornadaControllerTests" name="testUpdate" time="0.291">
          <failure message="Assertion failed: &#xa;&#xa;assert jornada.save() != null&#xa;       |       |      |&#xa;       |       null   false&#xa;       com.climaescolar.Jornada : (unsaved)&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert jornada.save() != null
       |       |      |
       |       null   false
       com.climaescolar.Jornada : (unsaved)

	at com.climaescolar.JornadaControllerTests.testUpdate(JornadaControllerTests.groovy:100)
</failure>

      </testcase>

      <testcase classname="com.climaescolar.JornadaControllerTests" name="testEdit" time="0.047">
          <failure message="Assertion failed: &#xa;&#xa;assert jornada.save() != null&#xa;       |       |      |&#xa;       |       null   false&#xa;       com.climaescolar.Jornada : (unsaved)&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert jornada.save() != null
       |       |      |
       |       null   false
       com.climaescolar.Jornada : (unsaved)

	at com.climaescolar.JornadaControllerTests.testEdit(JornadaControllerTests.groovy:80)
</failure>

      </testcase>

      <testcase classname="com.climaescolar.JornadaControllerTests" name="testDelete" time="0.045">
          <failure message="Assertion failed: &#xa;&#xa;assert jornada.save() != null&#xa;       |       |      |&#xa;       |       null   false&#xa;       com.climaescolar.Jornada : (unsaved)&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert jornada.save() != null
       |       |      |
       |       null   false
       com.climaescolar.Jornada : (unsaved)

	at com.climaescolar.JornadaControllerTests.testDelete(JornadaControllerTests.groovy:144)
</failure>

      </testcase>

      <testcase classname="com.climaescolar.JornadaControllerTests" name="testShow" time="0.041">
          <failure message="Assertion failed: &#xa;&#xa;assert jornada.save() != null&#xa;       |       |      |&#xa;       |       null   false&#xa;       com.climaescolar.Jornada : (unsaved)&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert jornada.save() != null
       |       |      |
       |       null   false
       com.climaescolar.Jornada : (unsaved)

	at com.climaescolar.JornadaControllerTests.testShow(JornadaControllerTests.groovy:62)
</failure>

      </testcase>

      <testcase classname="com.climaescolar.JornadaControllerTests" name="testCreate" time="0.034" />

      <testcase classname="com.climaescolar.JornadaControllerTests" name="testList" time="0.037" />

      <testcase classname="com.climaescolar.JornadaControllerTests" name="testIndex" time="0.037" />

      <testcase classname="com.climaescolar.JornadaControllerTests" name="testSave" time="0.051">
          <failure message="Assertion failed: &#xa;&#xa;assert response.redirectedUrl == &apos;/jornada/show/1&apos;&#xa;       |        |             |&#xa;       |        null          false&#xa;       org.codehaus.groovy.grails.plugins.testing.GrailsMockHttpServletResponse@7c995227&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert response.redirectedUrl == &apos;/jornada/show/1&apos;
       |        |             |
       |        null          false
       org.codehaus.groovy.grails.plugins.testing.GrailsMockHttpServletResponse@7c995227

	at com.climaescolar.JornadaControllerTests.testSave(JornadaControllerTests.groovy:48)
</failure>

      </testcase>

      <system-out><![CDATA[--Output from testUpdate--
--Output from testEdit--
--Output from testDelete--
--Output from testShow--
--Output from testCreate--
--Output from testList--
--Output from testIndex--
--Output from testSave--
]]></system-out>

      <system-err><![CDATA[--Output from testUpdate--
--Output from testEdit--
--Output from testDelete--
--Output from testShow--
--Output from testCreate--
--Output from testList--
--Output from testIndex--
--Output from testSave--
]]></system-err>

  </testsuite>
  <testsuite errors="0" failures="5" hostname="ROGG46" id="10" name="OpcionControllerTests" package="com.climaescolar" tests="8" time="1.087" timestamp="2015-07-23T05:50:22">
      <properties />

      <testcase classname="com.climaescolar.OpcionControllerTests" name="testUpdate" time="0.294">
          <failure message="Assertion failed: &#xa;&#xa;assert opcion.save() != null&#xa;       |      |      |&#xa;       |      null   false&#xa;       com.climaescolar.Opcion : (unsaved)&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert opcion.save() != null
       |      |      |
       |      null   false
       com.climaescolar.Opcion : (unsaved)

	at com.climaescolar.OpcionControllerTests.testUpdate(OpcionControllerTests.groovy:100)
</failure>

      </testcase>

      <testcase classname="com.climaescolar.OpcionControllerTests" name="testEdit" time="0.041">
          <failure message="Assertion failed: &#xa;&#xa;assert opcion.save() != null&#xa;       |      |      |&#xa;       |      null   false&#xa;       com.climaescolar.Opcion : (unsaved)&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert opcion.save() != null
       |      |      |
       |      null   false
       com.climaescolar.Opcion : (unsaved)

	at com.climaescolar.OpcionControllerTests.testEdit(OpcionControllerTests.groovy:80)
</failure>

      </testcase>

      <testcase classname="com.climaescolar.OpcionControllerTests" name="testDelete" time="0.41">
          <failure message="Assertion failed: &#xa;&#xa;assert opcion.save() != null&#xa;       |      |      |&#xa;       |      null   false&#xa;       com.climaescolar.Opcion : (unsaved)&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert opcion.save() != null
       |      |      |
       |      null   false
       com.climaescolar.Opcion : (unsaved)

	at com.climaescolar.OpcionControllerTests.testDelete(OpcionControllerTests.groovy:144)
</failure>

      </testcase>

      <testcase classname="com.climaescolar.OpcionControllerTests" name="testShow" time="0.042">
          <failure message="Assertion failed: &#xa;&#xa;assert opcion.save() != null&#xa;       |      |      |&#xa;       |      null   false&#xa;       com.climaescolar.Opcion : (unsaved)&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert opcion.save() != null
       |      |      |
       |      null   false
       com.climaescolar.Opcion : (unsaved)

	at com.climaescolar.OpcionControllerTests.testShow(OpcionControllerTests.groovy:62)
</failure>

      </testcase>

      <testcase classname="com.climaescolar.OpcionControllerTests" name="testCreate" time="0.034" />

      <testcase classname="com.climaescolar.OpcionControllerTests" name="testList" time="0.034" />

      <testcase classname="com.climaescolar.OpcionControllerTests" name="testIndex" time="0.035" />

      <testcase classname="com.climaescolar.OpcionControllerTests" name="testSave" time="0.042">
          <failure message="Assertion failed: &#xa;&#xa;assert response.redirectedUrl == &apos;/opcion/show/1&apos;&#xa;       |        |             |&#xa;       |        null          false&#xa;       org.codehaus.groovy.grails.plugins.testing.GrailsMockHttpServletResponse@77dfc020&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert response.redirectedUrl == &apos;/opcion/show/1&apos;
       |        |             |
       |        null          false
       org.codehaus.groovy.grails.plugins.testing.GrailsMockHttpServletResponse@77dfc020

	at com.climaescolar.OpcionControllerTests.testSave(OpcionControllerTests.groovy:48)
</failure>

      </testcase>

      <system-out><![CDATA[--Output from testUpdate--
--Output from testEdit--
--Output from testDelete--
--Output from testShow--
--Output from testCreate--
--Output from testList--
--Output from testIndex--
--Output from testSave--
]]></system-out>

      <system-err><![CDATA[--Output from testUpdate--
--Output from testEdit--
--Output from testDelete--
--Output from testShow--
--Output from testCreate--
--Output from testList--
--Output from testIndex--
--Output from testSave--
]]></system-err>

  </testsuite>
  <testsuite errors="1" failures="4" hostname="ROGG46" id="11" name="OperarioControllerTests" package="com.climaescolar" tests="8" time="0.908" timestamp="2015-07-23T05:50:23">
      <properties />

      <testcase classname="com.climaescolar.OperarioControllerTests" name="testUpdate" time="0.338">
          <failure message="Assertion failed: &#xa;&#xa;assert operario.save() != null&#xa;       |        |      |&#xa;       |        null   false&#xa;       com.climaescolar.Operario : (unsaved)&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert operario.save() != null
       |        |      |
       |        null   false
       com.climaescolar.Operario : (unsaved)

	at com.climaescolar.OperarioControllerTests.testUpdate(OperarioControllerTests.groovy:100)
</failure>

      </testcase>

      <testcase classname="com.climaescolar.OperarioControllerTests" name="testEdit" time="0.086">
          <failure message="Assertion failed: &#xa;&#xa;assert operario.save() != null&#xa;       |        |      |&#xa;       |        null   false&#xa;       com.climaescolar.Operario : (unsaved)&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert operario.save() != null
       |        |      |
       |        null   false
       com.climaescolar.Operario : (unsaved)

	at com.climaescolar.OperarioControllerTests.testEdit(OperarioControllerTests.groovy:80)
</failure>

      </testcase>

      <testcase classname="com.climaescolar.OperarioControllerTests" name="testDelete" time="0.073">
          <failure message="Assertion failed: &#xa;&#xa;assert operario.save() != null&#xa;       |        |      |&#xa;       |        null   false&#xa;       com.climaescolar.Operario : (unsaved)&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert operario.save() != null
       |        |      |
       |        null   false
       com.climaescolar.Operario : (unsaved)

	at com.climaescolar.OperarioControllerTests.testDelete(OperarioControllerTests.groovy:144)
</failure>

      </testcase>

      <testcase classname="com.climaescolar.OperarioControllerTests" name="testShow" time="0.085">
          <failure message="Assertion failed: &#xa;&#xa;assert operario.save() != null&#xa;       |        |      |&#xa;       |        null   false&#xa;       com.climaescolar.Operario : (unsaved)&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert operario.save() != null
       |        |      |
       |        null   false
       com.climaescolar.Operario : (unsaved)

	at com.climaescolar.OperarioControllerTests.testShow(OperarioControllerTests.groovy:62)
</failure>

      </testcase>

      <testcase classname="com.climaescolar.OperarioControllerTests" name="testCreate" time="0.058" />

      <testcase classname="com.climaescolar.OperarioControllerTests" name="testList" time="0.057" />

      <testcase classname="com.climaescolar.OperarioControllerTests" name="testIndex" time="0.047" />

      <testcase classname="com.climaescolar.OperarioControllerTests" name="testSave" time="0.057">
          <error message="No signature of method: com.climaescolar.Role.methodMissing() is applicable for argument types: () values: []" type="groovy.lang.MissingMethodException">groovy.lang.MissingMethodException: No signature of method: com.climaescolar.Role.methodMissing() is applicable for argument types: () values: []
	at com.climaescolar.OperarioController.save(OperarioController.groovy:26)
	at com.climaescolar.OperarioControllerTests.testSave(OperarioControllerTests.groovy:38)
</error>

      </testcase>

      <system-out><![CDATA[--Output from testUpdate--
--Output from testEdit--
--Output from testDelete--
--Output from testShow--
--Output from testCreate--
--Output from testList--
--Output from testIndex--
--Output from testSave--
]]></system-out>

      <system-err><![CDATA[--Output from testUpdate--
--Output from testEdit--
--Output from testDelete--
--Output from testShow--
--Output from testCreate--
--Output from testList--
--Output from testIndex--
--Output from testSave--
]]></system-err>

  </testsuite>
  <testsuite errors="0" failures="1" hostname="ROGG46" id="12" name="OperarioTests" package="com.climaescolar" tests="1" time="0.237" timestamp="2015-07-23T05:50:24">
      <properties />

      <testcase classname="com.climaescolar.OperarioTests" name="testSomething" time="0.112">
          <failure message="junit.framework.AssertionFailedError: Implement me" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: Implement me
	at com.climaescolar.OperarioTests.testSomething(OperarioTests.groovy:15)
</failure>

      </testcase>

      <system-out><![CDATA[--Output from testSomething--
]]></system-out>

      <system-err><![CDATA[--Output from testSomething--
]]></system-err>

  </testsuite>
  <testsuite errors="0" failures="5" hostname="ROGG46" id="13" name="PreguntaControllerTests" package="com.climaescolar" tests="8" time="0.695" timestamp="2015-07-23T05:50:24">
      <properties />

      <testcase classname="com.climaescolar.PreguntaControllerTests" name="testUpdate" time="0.271">
          <failure message="Assertion failed: &#xa;&#xa;assert pregunta.save() != null&#xa;       |        |      |&#xa;       |        null   false&#xa;       com.climaescolar.Pregunta : (unsaved)&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert pregunta.save() != null
       |        |      |
       |        null   false
       com.climaescolar.Pregunta : (unsaved)

	at com.climaescolar.PreguntaControllerTests.testUpdate(PreguntaControllerTests.groovy:100)
</failure>

      </testcase>

      <testcase classname="com.climaescolar.PreguntaControllerTests" name="testEdit" time="0.037">
          <failure message="Assertion failed: &#xa;&#xa;assert pregunta.save() != null&#xa;       |        |      |&#xa;       |        null   false&#xa;       com.climaescolar.Pregunta : (unsaved)&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert pregunta.save() != null
       |        |      |
       |        null   false
       com.climaescolar.Pregunta : (unsaved)

	at com.climaescolar.PreguntaControllerTests.testEdit(PreguntaControllerTests.groovy:80)
</failure>

      </testcase>

      <testcase classname="com.climaescolar.PreguntaControllerTests" name="testDelete" time="0.034">
          <failure message="Assertion failed: &#xa;&#xa;assert pregunta.save() != null&#xa;       |        |      |&#xa;       |        null   false&#xa;       com.climaescolar.Pregunta : (unsaved)&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert pregunta.save() != null
       |        |      |
       |        null   false
       com.climaescolar.Pregunta : (unsaved)

	at com.climaescolar.PreguntaControllerTests.testDelete(PreguntaControllerTests.groovy:144)
</failure>

      </testcase>

      <testcase classname="com.climaescolar.PreguntaControllerTests" name="testShow" time="0.038">
          <failure message="Assertion failed: &#xa;&#xa;assert pregunta.save() != null&#xa;       |        |      |&#xa;       |        null   false&#xa;       com.climaescolar.Pregunta : (unsaved)&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert pregunta.save() != null
       |        |      |
       |        null   false
       com.climaescolar.Pregunta : (unsaved)

	at com.climaescolar.PreguntaControllerTests.testShow(PreguntaControllerTests.groovy:62)
</failure>

      </testcase>

      <testcase classname="com.climaescolar.PreguntaControllerTests" name="testCreate" time="0.037" />

      <testcase classname="com.climaescolar.PreguntaControllerTests" name="testList" time="0.036" />

      <testcase classname="com.climaescolar.PreguntaControllerTests" name="testIndex" time="0.042" />

      <testcase classname="com.climaescolar.PreguntaControllerTests" name="testSave" time="0.051">
          <failure message="Assertion failed: &#xa;&#xa;assert response.redirectedUrl == &apos;/pregunta/show/1&apos;&#xa;       |        |             |&#xa;       |        null          false&#xa;       org.codehaus.groovy.grails.plugins.testing.GrailsMockHttpServletResponse@264aa8a3&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert response.redirectedUrl == &apos;/pregunta/show/1&apos;
       |        |             |
       |        null          false
       org.codehaus.groovy.grails.plugins.testing.GrailsMockHttpServletResponse@264aa8a3

	at com.climaescolar.PreguntaControllerTests.testSave(PreguntaControllerTests.groovy:48)
</failure>

      </testcase>

      <system-out><![CDATA[--Output from testUpdate--
--Output from testEdit--
--Output from testDelete--
--Output from testShow--
--Output from testCreate--
--Output from testList--
--Output from testIndex--
--Output from testSave--
]]></system-out>

      <system-err><![CDATA[--Output from testUpdate--
--Output from testEdit--
--Output from testDelete--
--Output from testShow--
--Output from testCreate--
--Output from testList--
--Output from testIndex--
--Output from testSave--
]]></system-err>

  </testsuite>
  <testsuite errors="0" failures="5" hostname="ROGG46" id="14" name="RespuestaControllerTests" package="com.climaescolar" tests="8" time="0.66" timestamp="2015-07-23T05:50:25">
      <properties />

      <testcase classname="com.climaescolar.RespuestaControllerTests" name="testUpdate" time="0.364">
          <failure message="Assertion failed: &#xa;&#xa;assert respuesta.save() != null&#xa;       |         |      |&#xa;       |         null   false&#xa;       com.climaescolar.Respuesta : (unsaved)&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert respuesta.save() != null
       |         |      |
       |         null   false
       com.climaescolar.Respuesta : (unsaved)

	at com.climaescolar.RespuestaControllerTests.testUpdate(RespuestaControllerTests.groovy:100)
</failure>

      </testcase>

      <testcase classname="com.climaescolar.RespuestaControllerTests" name="testEdit" time="0.048">
          <failure message="Assertion failed: &#xa;&#xa;assert respuesta.save() != null&#xa;       |         |      |&#xa;       |         null   false&#xa;       com.climaescolar.Respuesta : (unsaved)&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert respuesta.save() != null
       |         |      |
       |         null   false
       com.climaescolar.Respuesta : (unsaved)

	at com.climaescolar.RespuestaControllerTests.testEdit(RespuestaControllerTests.groovy:80)
</failure>

      </testcase>

      <testcase classname="com.climaescolar.RespuestaControllerTests" name="testDelete" time="0.037">
          <failure message="Assertion failed: &#xa;&#xa;assert respuesta.save() != null&#xa;       |         |      |&#xa;       |         null   false&#xa;       com.climaescolar.Respuesta : (unsaved)&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert respuesta.save() != null
       |         |      |
       |         null   false
       com.climaescolar.Respuesta : (unsaved)

	at com.climaescolar.RespuestaControllerTests.testDelete(RespuestaControllerTests.groovy:144)
</failure>

      </testcase>

      <testcase classname="com.climaescolar.RespuestaControllerTests" name="testShow" time="0.048">
          <failure message="Assertion failed: &#xa;&#xa;assert respuesta.save() != null&#xa;       |         |      |&#xa;       |         null   false&#xa;       com.climaescolar.Respuesta : (unsaved)&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert respuesta.save() != null
       |         |      |
       |         null   false
       com.climaescolar.Respuesta : (unsaved)

	at com.climaescolar.RespuestaControllerTests.testShow(RespuestaControllerTests.groovy:62)
</failure>

      </testcase>

      <testcase classname="com.climaescolar.RespuestaControllerTests" name="testCreate" time="0.044" />

      <testcase classname="com.climaescolar.RespuestaControllerTests" name="testList" time="0.034" />

      <testcase classname="com.climaescolar.RespuestaControllerTests" name="testIndex" time="0.033" />

      <testcase classname="com.climaescolar.RespuestaControllerTests" name="testSave" time="0.039">
          <failure message="Assertion failed: &#xa;&#xa;assert response.redirectedUrl == &apos;/respuesta/show/1&apos;&#xa;       |        |             |&#xa;       |        null          false&#xa;       org.codehaus.groovy.grails.plugins.testing.GrailsMockHttpServletResponse@213d95be&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert response.redirectedUrl == &apos;/respuesta/show/1&apos;
       |        |             |
       |        null          false
       org.codehaus.groovy.grails.plugins.testing.GrailsMockHttpServletResponse@213d95be

	at com.climaescolar.RespuestaControllerTests.testSave(RespuestaControllerTests.groovy:48)
</failure>

      </testcase>

      <system-out><![CDATA[--Output from testUpdate--
--Output from testEdit--
--Output from testDelete--
--Output from testShow--
--Output from testCreate--
--Output from testList--
--Output from testIndex--
--Output from testSave--
]]></system-out>

      <system-err><![CDATA[--Output from testUpdate--
--Output from testEdit--
--Output from testDelete--
--Output from testShow--
--Output from testCreate--
--Output from testList--
--Output from testIndex--
--Output from testSave--
]]></system-err>

  </testsuite>
</testsuites>
